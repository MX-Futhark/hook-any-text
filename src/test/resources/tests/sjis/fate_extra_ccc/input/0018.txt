93F1906C8163816382A282E2814193F191CC814200000000000000000001000000020000000300000004000000050000000600000007000000080000000900000B0000000000000000000000000000000000000000001F01000000000000000000000000FFFFFFFF00002A430000B841000000000000604100000000FFFFFFFF000038430000B841000000000000000000006041FFFFFFFF00002A4300001442000000000000604100006041FFFFFFFF00003843000014420000000090F0A248020000003814C908000000000000000000000000FFFFFFFF0000000058EFA248020000005814C908000000000000000000000000FFFFFFFF00010000C0EFA248020000007814C908000000000000000000000000FFFFFFFF0001000028F0A248020000009814C908000000000000000000000000FFFFFFFF0001000090F0A2480200000000000000000000000000000000000000FFFFFFFF00010000F8F0A24802000000D814C908000000000000000000000000030001000000000060F1A24802000000F814C9080000000000000000000000000300010000000000C8F1A248020000001815C908000000000000000000000000030001000000000030F2A248020000003815C908000000000000000000000000030001000000000098F2A248020000005815C908000000000000000000000000030001000000000000F3A248020000007815C908000000000000000000000000030001000000000068F3A248020000009815C9080000000000000000000000000300010000000000D0F3A24802000000B815C908000000000000000000000000030001000000000038F4A24802000000D815C9080000000000000000000000000300010000000000A0F4A24802000000F815C908000000000000000000000000030001000000000008F5A248020000001816C908000000000000000000000000030001000000000070F5A248020000003816C908000000000000000000