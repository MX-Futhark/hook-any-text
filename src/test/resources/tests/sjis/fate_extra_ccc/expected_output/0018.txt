["二人……いや、二体。"]